tendermint/spn
# feat(`monitoringc`): add `CheckValidatorSet` in `MsgCreateClient`

Complete the `MsgCreateClient` message by adding the check for the provided validator set the for the IBC client to verify

The message fails if the check fails

Add tests for the message handler

To test the feature:
- Follow the monitoring guide with `orbit`
- When `orbit` is running, add a genesis validator on `spn` with the consensus key of the validator on `orbit` and trigger the launch to simulate a launch with orbit chain
- `MsgCreateClient` should fail if not the right validator set with the proper consensus key is provided
